README.md

```diff
@@ -36,6 +36,22 @@ $ ccm start
 # Wait a lot...
 ```
 
+### Create a cluster of nodes with a specific build-id
+
+Let's say you want to create a cluster of `Scylla 2023.1.0~rc8-20230731.b6f7c5a6910c`, with build id `f6e718548e76ccf3564ed2387b6582ba8d37793c`.
+
+1. Go to https://backtrace.scylladb.com and find your desired Scylla version
+2. Click the arrow down symbol (\\/) to show all available download links
+3. Download the unified Scylla package (`unified-pack-url-x86_64`)
+4. Create a 3 node cluster:
+```bash
+# The unified package will be extracted to ~/.ccm/scylla-repository/my_custom_scylla
+# Make sure that the version name (my_custom_scylla) is different for each unified package you use, otherwise ccm will use the previously extracted version. 
+ccm create my_cluster -n 3 --scylla --vnodes \
+    --version my_custom_scylla \
```

Comment:
Wow. If you don't invent this name "my_custom_scylla", can't it just use the full name of the downloaded version? :-(

Reply:
Without the `version` argument it fails with:
```bash
ccm: error: ./ is not a valid cassandra directory. You must define a cassandra dir or version.
```

I don't know if there's a better way to do it. I found this one in the `dtest` docs I think:
https://github.com/scylladb/scylla-dtest/blob/3fe7961e06ff02c72000b8c162a3838185456a39/README.md?plain=1#L204